ensime-sbt.el

```diff
@@ -53,6 +53,13 @@
   (interactive)
   (sbt-command "test:compile"))
 
+(defun ensime-sbt-do-compile-only ()
+  (interactive)
+  (let ((subproject (ensime-sbt-find-subproject buffer-file-name)))
+    (if subproject
+        (sbt-command (concat subproject "/ensimeCompileOnly " buffer-file-name))
```

Comment:
It might make it easier to test this code if the calculation of the command was separated from the running of it with sbt-command (you'd need to provide a .ensime file and probably mock the connection). That would mean you wouldn't need to write a big slow integration test.